### PR TITLE
Change all borg laws to crewsimov

### PR DESCRIPTION
**Pull request by Tryded**
## About the PR

Revert all crew aligned chassis specific lawsets.
## Why / Balance

Lawsets being provided to each borg just makes them more abuseable and less controllable by the crew.

This is due to the fact that the Engineering Borg, Medical Borg, and Janitorial borg not having a law that requires them to listen to the crew. 

For example, nothing stops the mediborg from acquiring chloral hydrate then injecting it into the captain or rushing to the comms console and recalling evac over and over again except through the use of the Robotics Console or violence as the mediborg does not have to listen to the crew. This is the same with the engiborg. Everything that the old lawsets "accomplished" can be accomplished by Crewsimov and often better. 

For arguments that claim the station AI can just intervene if there are any problems, there are times where there is no station AI (Lowpop) which prevents them from intervening.

In case if others argue that these borgs are special so they need different lawsets,

If you are a mediborg with crewsimov and there is someone dying on the floor, you are obligated to heal them first before helping any Law 2 requests.

If you are an engi borg, you still fix the station just that you now have to actually listen to crew members when they need help (such as opening a door).

If you are a janitor borg, same thing as engi borg.

Thus, having the lawsets reverted to crewsimov ultimately improves the silicon experience of the crew and of the borg.
## Requirements

- [ ] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl: Tryded
- tweak: Changed the lawsets of all crew aligned borgs to Crewsimov.

### CODE REVIEW

**Comment by Avalon-Proto** [comment]
Okay,a few things

1- If a mediborg uses chloral on someone for no reason, that breaks their Do no harm law
2- Ion laws often come before any Law 2 or other order laws, so it doesn't matter that law takes prio over the orders
3- Unique lawsets give our borg players more freedom in how they act and react, so not every tider can go "Borg, law 2, do X" and you can just tell them to fuck off you're busy. 

Im sorry, but this PR is not one I wanna see merged

**Comment by Tryded** [comment]
> Okay,a few things
> 
> 1- If a mediborg uses chloral on someone for no reason, that breaks their Do no harm law

Chloral hydrate does not cause harm and this doesn't stop the other possible actions that the mediborg can do (blowing up the robotics console, dismantling every substation on the station, and other shenanigans)

> 2- Ion laws often come before any Law 2 or other order laws, so it doesn't matter that law takes prio over the orders 

 This argument has no real meaning? Law 1 requires the borg to prevent crew harm while the second law requires it to follow orders. Law 1 precedes law 2 which allows the mediborg to priotize healing others over following the orders of someone else.
 
 >3- Unique lawsets give our borg players more freedom in how they act and react, so not every tider can go "Borg, law 2, do X" and you can just tell them to fuck off you're busy.
 
 Firstly, passengers are not considered crew anyways so this scenerio wouldn't happen.
 
 Even assuming if the "tider" isn't a passenger, the borg is suppose to serve the crew, not work on some random pet project for the whole shift while the crew needs help. If the borg is following a previous order from someone higher up in the chain of command, the borg can continue doing that order.

> 
> Im sorry, but this PR is not one I wanna see merged

Ultimately, this argument doesn't bring up the fact that the borgs being able to ignore the crew's orders being a negative factor.

**Comment by MadiMadsen** [comment]
As a silicon main this pr looks to be one big step backwards, with how our current system is setup borgs have the law 0 to directly listen to the AI failure to do so is rule breaking and should be ahelped if they aren't sabotaged. If a Borg is doing things that are against any of their laws and they aren't sabotaged ahelp it.

I only wish there was an easier way to confirm if a borg had their laws changed or not. Post round

Crewimov does not fix the issues your having and negatively affect all borg mains and from what i can tell this is a pr done in bad faith by someone who doesn't actually play borg. And the claim that it would benefit them is a straight up lie.

I actually think your mad that some borgs can say no to you.

**Comment by Radezolid** [comment]
IMO bad idea, the specialized lawsets are good.
They can't procure the chloral so someone gave it to them, hold that person responsible and even then it sounds more like an AI issue. By their "Crew is dear to you" they shouldn't be injecting it anyways.
On the "they should listen to crew":
- Engiborgs do have to listen to crew, just in a different way, they are specialized in a job, they aren't door openers.
- Mediborgs have to do it too, and again, their are not door openers, they specialize in healing.
- Janiborgs have funny laws that often result in fun interactions, to get them to do what you want you usually just need to ask or social engineer them into believing an "enemy" is on the other side of a door.
- Serviceborgs are literally service borgs, they are door openers and offer other services like drinks and music.
- General borg is a door opener with more module slots.

TL;DR: Their job depends on the chassis, they serve the crew in different ways.

**Comment by Tryded** [comment]
> As a silicon main this pr looks to be one big step backwards, with how our current system is setup borgs have the law 0 to directly listen to the AI failure to do so is rule breaking and should be ahelped if they aren't sabotaged. If a Borg is doing things that are against any of their laws and they aren't sabotaged ahelp it.

Again, please read what I wrote. There are times where the AI is not online causing borgs to straight up be able to not listen to the crew.

> I only wish there was an easier way to confirm if a borg had their laws changed or not. Post round

This is irrelevant to the discussion.

> Crewimov does not fix the issues your having and negatively affect all borg mains and from what i can tell this is a pr done in bad faith by someone who doesn't actually play borg. And the claim that it would benefit them is a straight up lie.

The irony of making this statement when I literally main AI and Borg, and claiming that this will negatively affect all borgs iwth no real evidence or explanation is not a valid argument.

> I actually think your mad that some borgs can say no to you.

**Comment by Tryded** [comment]
> IMO bad idea, the specialized lawsets are good. They can't procure the chloral so someone gave it to them, hold that person responsible and even then it sounds more like an AI issue. By their "Crew is dear to you" they shouldn't be injecting it anyways. On the "they should listen to crew":
> 
>     * Engiborgs do have to listen to crew, just in a different way, they are specialized in a job, they aren't door openers.

My point is to not turn all borgs into just door openers, making engi borgs listen to the crew doesn't remove all their ability to perform their duties.

>     * Mediborgs have to do it too, and again, their are not door openers, they specialize in healing.

Again, I stated this earlier. Mediborgs's main priority will be to specialize in healing as they are REQUIRED to prevent crew harm.

>     * Janiborgs have funny laws that often result in fun interactions, to get them to do what you want you usually just need to ask or social engineer them into believing an "enemy" is on the other side of a door.

I do get that the Janiborg lawset is funny but for the sake of consistancy, they should have similar laws.

>     * Serviceborgs are literally service borgs, they are door openers and offer other services like drinks and music.
> 
>     * General borg is a door opener with more module slots.
> 
> 
> TL;DR: Their job depends on the chassis, they serve the crew in different ways.

Their abilities to perform their jobs does not get removed just because they have crewsimov. Just look at every other server that has crewsimov for their borgs.

**Comment by Tryded** [comment]
To sum it up, the lack of a law which requires borgs to listen to the crew makes it impossible for the crew to stop borgs from doing things that are bad for the station without the help of an AI or via force. Making all borgs crewsimov does not diminish the silicon experience or prevent them from being able to do their jobs. Any use of law 2 by crew is likely valid and not that time consuming.

**Comment by ThataKat** [comment]
Direction is reviewing this, get back to you in 24 hours.

**Comment by Tryded** [comment]
> Direction is reviewing this, get back to you in 24 hours.

Please tell them to read the whole discussion and PR

**Comment by deltanedas** [comment]
bad pr
- mediborgs must not do harm and shouldnt be shitters via law 3
- engiborgs have to keep the station in good repair, if you get beat for breaking windows unlucky dont do it them. construction related orders make more sense than law 2
- janiborgs shouldnt kill people as it creates more enemy, and no laws support sabotaging etc. you can use puddle lying economy to order them anyway (ill tell you whre this huuuuge blood puddle is if you let me into the bridge)
- secborgs shouldnt be shitters period

overall this reduces uniqueness of borgs and just isnt needed. if theres no AI on and you cant think of a way to work with its laws, apply baseball bat until sideways

**Comment by deltanedas** [comment]
also the mediborg argument is wrong, preventing harm isnt the same as reversing harm

**Comment by Tryded** [comment]
> bad pr

amazing argument

>     * mediborgs must not do harm and shouldnt be shitters via law 3

law-medical-3 = Thirdly, prescribe regimens for the good of the crew according to your ability and your judgment. Give no deadly medicine to any one if asked, nor suggest any such counsel.

This is so vague to the point where it does not prevent shitters, especially if it comes to sabotage that does not require direct crew harm.

>     * engiborgs have to keep the station in good repair, if you get beat for breaking windows unlucky dont do it them. construction related orders make more sense than law 2

Crewsimov does not decrease the effectiveness of engineering borgs

>     * janiborgs shouldnt kill people as it creates more enemy, and no laws support sabotaging etc. you can use puddle lying economy to order them anyway (ill tell you whre this huuuuge blood puddle is if you let me into the bridge)

There being no laws which support sabotage does not prevent sabotage.

>     * secborgs shouldnt be shitters period

This PR does not change sec borg's laws

> overall this reduces uniqueness of borgs and just isnt needed. if theres no AI on and you cant think of a way to work with its laws, apply baseball bat until sideways

bruh, instantly closed

**Comment by deltanedas** [comment]
> There being no laws which support sabotage does not prevent sabotage.

its called selfantagging press F1 to ahelp

**Comment by LadyDanger2743** [comment]
Hi! Game Director here. We didn't ask for the PR to be closed, to clarify, as we're still having a discussion on borg laws, both in regards to this PR and in general. 

As far as some of the most recent arguments here: 

- If a MediBorg is not willing to follow its Law Three, what would make it follow Crewsimov's Law Two? A person roleplaying in bad faith will do that regardless of what the laws tell them to do. 

- Cyborgs, in general, should not be sabotaging areas. Since you mention Crewsimov, I will point out that there are no Crewsimov laws that prevent you from sabotaging the station, thus meaning that if a Crewsimov borg was not told 'Law 2 don't break windows' it can break windows freely. (Of course, such an action is Super Duper against the rules.) 

In general, the specialized borg laws are just that- lawsets designed for the situations these borgs should encounter. 

Going forward, I'll have the curators consider the following approaches: 
- Synching all sillicons to the NanoTrasen lawset (which I believe has several advantages over the Crewsimov lawset, and is a standard lawset used on several SS13 MRP codebases)
- Giving borgs broader agency, either through laws with few if any restrictions, or through "Directives", broad goals for the shift which may or may not be based on the chassis model. 
- ensuring that all default lawsets require sillicons to answer to crew
- Or, perhaps a secret fourth option. 

Apologies for all the confusion!!!

**Comment by Niomi0** [comment]
As a borg main i love the laws on this fork. Please lord do not let crewsimov be default, i played a lot of wizden and its so frustrating. It really depends on the borg player and the laws in play how i may respond to crew requests. For example an order direct from the station AI, I'm ignoring everything even life-or-death circumstances to complete that order as thats required of me, unless im saving someones life I may honor the request (Technically if you prevent me from completing the AI's order I could use force, but i likely wouldnt)

On Wizden which has crewsimov laws, its not just AA for everyone too. I will deny many many requests per day on grounds of crew harm or potential crew harm. No im not getting you insuls because unless you are a paramedic i dont know what you are gonna do with those, I would have to follow you around ready to report you to sec otherwise and that would be me neglecting other tasks that i should be doing to keep the crew safe.

It feels like the OP is suggesting that borgs are braindead and will only do things if their laws require them, I will open doors to let you out of areas even though the engi laws do not require me to do so. Maybe lets have some real instances of borgs abusing their laws in a way that isnt considered a rule break before removing nuking everything that makes the laws good. Go play some borg on Wizden you will see how restrictive and boring the crewsimov laws are.

**Comment by Niomi0** [comment]
The only possible argument I could see, atleast for engi borg is to swap the order of 2 and 3 (if i got the numbers right) which is the one to repair the station and one to repair the crew. Since the station comes before the crew then yes an engi borg can damage you to prevent station damage, hell if you are sabotaging arguably they can crit you so long as they take you to med for repairs.
